CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,5 @@
 - Add: NGSIv2 support for Timestamp Compression plugin, through configuration (#527)
 - Add: NGSIv2 support for Timestamp Processing plugin, through configuration (#527)
 - Add: NGSIv2 support for Expression Translation, Attribute Alias and Event plugins, through configuration (#527)
+- Add: NGSIv2 support for Bidirectionality plugin (#527)
```

Comment:
It seems task/multientityPluginNgsi2 (this PR) was branched from task/bidirectionalityPlugin (PR #4), so this PR mixes changes in both case.

After merging PR #4 and merging task/activeAttsNgsi2 into task/multientityPluginNgsi2, this PR will include only the changes regarding multientity plugin and would be reviewable.